src/dmd/expression.d

```diff
@@ -1722,6 +1722,16 @@ extern (C++) abstract class Expression : RootObject
         return e1;
     }
 
+    static Expression combine(Expression e1, Expression e2, Expression e3)
+    {
+        return combine(combine(e1, e2), e3);
+    }
+
+    static Expression combine(Expression e1, Expression e2, Expression e3, Expression e4)
```

Comment:
These two functions seem a little weird.  Can we make it a single variadic; maybe a template?

Reply:
It's obvious what they do with a quick glance. Using variadics or templates, not so easy.